### PR TITLE
DOC-742: Document changes to lists plugin with multiple text blocks selected

### PR DESCRIPTION
**Pull request by JamesToohey**
Related Ticket: DOC-742

Description of Changes:

Minor enhancement note for changes to `lists` plugin behaviour on selections across multiple text blocks.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] ~~`_data/nav.yml` has been updated (if applicable)~~
- [x] ~~Files has been included where required (if applicable)~~
- [x] ~~Files removed have been deleted, not just excluded from the build (if applicable)~~
- [x] ~~(New product features only) Release Note added~~

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes57.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ The following enhancements were made for the {{site.productname}} 5.7 release.
 
 {{site.productname}} 5.7 introduces the following minor enhancements:
 
-- changelog
+- The `lists` plugin will now apply list styles to all text blocks within a selection.
```
Comment (metricjs) [inline release-notes/release-notes57.md:46]: Might it be worth mentioning that it applies to mixed paragraph/list content? Really not sure

**Review by metricjs — APPROVED** [review]

**Review by lnewson — CHANGES_REQUESTED** [review]
As discussed during the call today, we want to call this out so it should be more of a "blurb" than a changelog entry.

**Review by EkimChau — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

File: release-notes/release-notes57.md [hunk 2]
```diff
@@ -37,7 +37,12 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
-### Enhancement name
+### Improved behavior when using `lists` plugin on mixed content
+
+Improved behavior when using the `lists` plugin to add lists to selections containing:
+
+- Multiple text blocks.
+- Mixes of existing lists and text blocks.
```
Comment (metricjs) [inline release-notes/release-notes57.md:45]:
```suggestion
- Combinations of existing lists and text blocks.
```
`Mixes` just reads weird to me. Not entirely sure this is better though... @tylerkelly13

**Review by metricjs — APPROVED** [review]

File: release-notes/release-notes57.md [hunk 3]
```diff
@@ -37,7 +37,12 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
-### Enhancement name
+### Improved behavior when using `lists` plugin on mixed content
+
+Improved behavior when using the `lists` plugin to add lists to selections containing:
```
Comment (tylerkelly13) [inline release-notes/release-notes57.md:42]:
```suggestion
The `lists` plugin has been updated to improve the handling of content containing:
```

Comment (tylerkelly13) [reply release-notes/release-notes57.md:42]: Trying to make it more of a complete sentence? It reads really weird at the moment. feels too much like a changelog entry.

**Review by tylerkelly13 — CHANGES_REQUESTED** [review]
please improve the description

**Review by ashfordneil — APPROVED** [review]

File: release-notes/release-notes57.md [hunk 4]
```diff
@@ -37,7 +37,12 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
-### Enhancement name
+### Improved behavior when using `lists` plugin on mixed content
+
+The `lists` plugin has been updated to improve the handling of content containing:
+
+- Multiple text blocks.
```
Comment (lnewson) [inline release-notes/release-notes57.md:44]: text blocks? That sounds confusing to me, as text can't be block elements.

File: release-notes/release-notes57.md [hunk 5]
```diff
@@ -37,7 +37,12 @@ The following new features were added for the {{site.productname}} 5.7 release.
 
 The following enhancements were made for the {{site.productname}} 5.7 release.
 
-### Enhancement name
+### Improved behavior when using `lists` plugin on mixed content
+
+The `lists` plugin has been updated to improve the handling of content containing:
```
Comment (lnewson) [inline release-notes/release-notes57.md:42]: Should this say something along the lines of "when converting content to lists"? As atm it's not very clear what "handling" is referring to here.

**Review by lnewson — APPROVED** [review]

**Review by tylerkelly13 — APPROVED** [review]

**Pull request merged**